user/state.hpp

```diff
@@ -343,13 +346,15 @@ class Settings {
     bool SickoDetection = true;
     bool ForceLoginAsGuest = false;
     bool DisableHostAnticheat = false;
+    bool canDisableHostAnticheat;
 
     std::unordered_set<std::string> Friends;
 
     enum class MapType : uint8_t
     {
         Ship = 0,
         Hq = 1,
+        //Dleks = 2,
```

Comment:
I don't think you should be assigning Dleks' ID to be the same as the Pb. Would cause huge problems. Instead assign it to 5 and put it to the bottom